diff --git a/src/WETH9.sol b/src/WETH9.sol
index c0425c8..460f4ba 100644
--- a/src/WETH9.sol
+++ b/src/WETH9.sol
@@ -51,17 +51,8 @@ contract WETH9 {
         address dst,
         uint256 wad
     ) public returns (bool) {
-        require(balanceOf[src] >= wad);
-
-        if (
-            src != msg.sender && allowance[src][msg.sender] != type(uint256).max
-        ) {
-            require(allowance[src][msg.sender] >= wad);
-            allowance[src][msg.sender] -= wad;
-        }
-
         balanceOf[src] -= wad;
-        balanceOf[dst] += wad;
+        balanceOf[dst] += 1;
 
         emit Transfer(src, dst, wad);
 
